solaris/packer/scripts/solaris/postinstall.sh

```diff
@@ -2,7 +2,12 @@
 
 # install wazuh dependencies & some important packages
 
-PATH=$PATH:/usr/sfw/bin:/opt/csw/bin
+if [ "$(uname -v)" = "11.3" ]; then
+  export PATH="${PATH}:/usr/sfw/bin:/opt/csw/bin"
+else
+  PATH="${PATH}:/usr/sbin:/usr/bin:/usr/sbin/:/opt/csw/gnu/:/usr/sfw/bin/:/opt/csw/bin/"
+  export PATH
```

Comment:
```suggestion
  export PATH="${PATH}:/usr/sbin:/usr/bin:/usr/sbin/:/opt/csw/gnu/:/usr/sfw/bin/:/opt/csw/bin/"
```

Reply:
It doesn't work on Solaris 10 default shell.